vscode-dotnet-runtime-library/src/test/unit/DotnetCoreAcquisitionWorker.test.ts

```diff
@@ -73,7 +74,7 @@ suite('DotnetCoreAcquisitionWorker Unit Tests', function ()
     function setupWorker(workerContext: IAcquisitionWorkerContext, eventStream: IEventStream): [MockDotnetCoreAcquisitionWorker, IAcquisitionInvoker]
     {
         const acquisitionWorker = getMockAcquisitionWorker(workerContext);
-        const invoker = new NoInstallAcquisitionInvoker(eventStream, acquisitionWorker);
+        const invoker = new NoInstallAcquisitionInvoker(eventStream, acquisitionWorker, workerContext, path.dirname(getExpectedPath(getInstallFromContext(workerContext).installId, workerContext.acquisitionContext.mode ?? 'runtime')));

```

Comment:
nit: Usually when nesting multiple method/function calls into a method/function call, I like to separate out the values just to make it easier to read. I know this is in test code, but as a general concept/process, it is good to follow.
```suggestion
        const installId = path.dirname(getExpectedPath(getInstallFromContext(workerContext).installId;
        const mode = workerContext.acquisitionContext.mode ?? 'runtime';
        const invoker = new NoInstallAcquisitionInvoker(eventStream, acquisitionWorker, workerContext, installId, mode)));
```